teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md

```diff
@@ -66,7 +66,7 @@ The phone number to unnassign from the user or resource account. Supports E.164
 
 ```yaml
 Type: System.String
-Parameter Sets: (All)
+Parameter Sets: (RemoveSome)

```

Comment:
This parameter set does not seem to exist in the _sintax_ section

Reply:
It seems this is the current sintax with the correct parameter sets

![image](https://user-images.githubusercontent.com/36666927/141126841-2f0f4624-9b08-48ce-9cd9-276739975c73.png)